### DISCUSSION TITLE
mastadon.social

### DISCUSSION THREAD

**Discussion by crazyvetfam** [mastodon/mastodon#19461] [Q&A] [answered]
says  they have closed the connection  is it just down or something going on when i try the website

**Comment by markcellus** [comment, marked as answer]
It's working for me. Are you sure you're typing the right website? You put mastadon.social in the post title. But it's mastodon.social.

**Reply by crazyvetfam** [reply]
![image](https://user-images.githubusercontent.com/88051046/198154777-919ffa28-593a-4637-9559-8fbe19b32797.png)
 maybe that worked yea it use to come up all the time
